source/HL_spi.c

```diff
@@ -418,14 +418,14 @@ void spiInit(void)
                    | (uint32)((uint32)0U << 0U);  /* C2EDELAY */
 
     /** - Data Format 0 */
-    spiREG3->FMT0 = (uint32)((uint32)0U << 24U)  /* wdelay */
+    spiREG3->FMT0 = (uint32)((uint32)2U << 24U)  /* wdelay */
                   | (uint32)((uint32)0U << 23U)  /* parity Polarity */
                   | (uint32)((uint32)0U << 22U)  /* parity enable */
                   | (uint32)((uint32)0U << 21U)  /* wait on enable */
                   | (uint32)((uint32)0U << 20U)  /* shift direction */
-                  | (uint32)((uint32)1U << 17U)  /* clock polarity */
+                  | (uint32)((uint32)0U << 17U)  /* clock polarity */
                   | (uint32)((uint32)0U << 16U)  /* clock phase */
-                  | (uint32)((uint32)2U << 8U) /* baudrate prescale */
+                  | (uint32)((uint32)74U << 8U) /* baudrate prescale */

```

Comment:
These SPI3 changes will affect SD cards on the dev boards. Perhaps using data fmt 1 for iris would work?